docs/pages/kubernetes-access/discovery/google-cloud.mdx

```diff
@@ -0,0 +1,464 @@
+forward user traffic to your GKE clusters. 
+
+Create a file called `GKEAccessManager.yaml` with the following content:
+
+```yaml
+title: GKE Cluster Access Manager
+description: "Manage access to GKE clusters"
+stage: GA
+includedPermissions:
+- container.clusters.impersonate
+- container.pods.get
+- container.selfSubjectAccessReviews.create
+- container.selfSubjectRulesReviews.create
+```
+
+Create the role, replacing `project-id` with the name of your project. If you
+receive a prompt indicating that certain permissions are in `TESTING`, enter
+`y`:
+
+```code
+$ gcloud iam roles create GKEAccessManager \
+  --project=project-id \
+  --file=GKEAccessManager.yaml
+```
+
+### Create a service account
+
+Now that you have declared roles for the Discovery Service and Kubernetes
+Service, create a service account so you can assign these roles.
+
+Run the following command to create a service account called
+`teleport-discovery-kubernetes`:
+
+```code
+$ gcloud iam service-accounts create teleport-discovery-kubernetes \
+  --description="Teleport Discovery Service and Kubernetes Service" \
+  --display-name="teleport-discovery-kubernetes"
+```
+
+Grant the roles you defined earlier to your service account, replacing
+`project-id` with the ID of your project.
+
+```code
+$ PROJECT_ID=project-id
+$ gcloud projects add-iam-policy-binding ${PROJECT_ID?} \
+   --member="serviceAccount:teleport-discovery-kubernetes@${PROJECT_ID?}.iam.gserviceaccount.com" \
+   --role="projects/${PROJECT_ID?}/roles/GKEKubernetesAutoDisc"
+$ gcloud projects add-iam-policy-binding ${PROJECT_ID?} \
+   --member="serviceAccount:teleport-discovery-kubernetes@${PROJECT_ID?}.iam.gserviceaccount.com" \
+   --role="projects/${PROJECT_ID?}/roles/GKEAccessManager"
+```
+
+<Details title="Deploying the Kubernetes Service and Discovery Service
+separately?">
+
+Create a service account for each service:
+
+```code
+$ gcloud iam service-accounts create teleport-discovery-service \
+  --description="Teleport Discovery Service" \
+  --display-name="teleport-discovery-service"
+$ gcloud iam service-accounts create teleport-kubernetes-service \
+  --description="Teleport Discovery Service" \
+  --display-name="teleport-discovery-service"
+```
+
+Grant the roles you defined earlier to your service accounts, replacing
+`project-id` with the ID of your project.
+
+```code
+$ PROJECT_ID=project-id
+$ gcloud projects add-iam-policy-binding ${PROJECT_ID?} \
+   --member="serviceAccount:teleport-discovery-service@${PROJECT_ID?}.iam.gserviceaccount.com" \
+   --role="projects/${PROJECT_ID?}/roles/GKEKubernetesAutoDisc"
+$ gcloud projects add-iam-policy-binding ${PROJECT_ID?} \
+   --member="serviceAccount:teleport-kubernetes-service@${PROJECT_ID?}.iam.gserviceaccount.com" \
+   --role="projects/${PROJECT_ID?}/roles/GKEAccessManager"
+```
+</Details>
+
+### Download a credentials file
+
+Download a credentials file for the service account used by the Discovery
```

Comment:
Make a notice block informing that this is not required if the goal is to run Teleport services in a GCP VM

Reply:
Would you still need to attach the service account to the GCP VM? I've added these instructions to attach the service account: https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#changeserviceaccountandscopes

Reply:
Yes